pyquil/job_results.py

```diff
@@ -100,7 +100,7 @@ def recover_complexes(coef_string, classical_addresses):
     """
     num_octets = len(coef_string)
     num_addresses = len(classical_addresses)
-    num_memory_octets = _round_to_next_multiple(num_addresses, 8) / 8
+    num_memory_octets = _round_to_next_multiple(num_addresses, 8) // 8
```

Comment:
Future bugs of this kind can be prevented with Mypy type annotation. While the Mypy static analysis can be done in Python 2, it is there by default since Python 3.5.

On a related note, WDYT of pledging to drop support for Python 2.7 no later than 2020 in http://www.python3statement.org/ ?